x-pack/plugins/alerting/server/alerts_service/lib/set_alerts_to_untracked.test.ts

```diff
@@ -426,6 +426,8 @@ describe('setAlertsToUntracked()', () => {
       },
     });
 
+    clusterClient.updateByQuery.mockResponseOnce({ total: 2, updated: 2 });
```

Comment:
We need to mock the response of `updateByQuery` for the test to test the correct behavior of the code.